pages/webview/content_page.py

```diff
@@ -22,15 +22,53 @@ def books_containing(self):
 
     class BooksContaining(Region):
         _root_locator = (By.CLASS_NAME, 'booksContaining')
+        _overview_locator = (By.CLASS_NAME, 'title')
         _book_list_locator = (By.CSS_SELECTOR, 'div > ul > li')
 
+        @property
+        def is_displayed(self):
+            return self.is_element_displayed(*self._book_list_locator)
+
         @property
         def book_list(self):
             return [self.Book(self.page, el) for el in self.find_elements(*self._book_list_locator)]
 
+        @property
```

Comment:
Why does this find one element but return a different one?

Reply:
I thought I needed to first go to the root_locator first which contains all the book lists, and then go down to overview_locator. I did not realize that it automatically goes to the root. 
So I got rid of the first line.